src/main/resources/db/migration/V1__imrt_ddl.sql

```diff
@@ -0,0 +1,70 @@
+/* Primary table containing all types of items and the json data from IAT */
```

Comment:
Are we going to add any information to the file name itself?  Or will all files just be named `V<number>__imrt_ddl.sql`?

Reply:
We can do either, I'd recommend always use the same name with a new version number. Let the checkin comments describe the change, rather than try and cram it into the filename.

Reply:
sgtm